### PR TITLE
HW8

### PR DESCRIPTION
**Pull request by MasiukBeTech**

### CODE REVIEW

File: lesson_8/index8.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <td>2021 - 2022</td>
+            <td>Technical support consultant <a href="https://blago.ua/" target="_blank">Blago</a></td>
+        </tr>
+        <tr>
+            <td>2022 - 2023</td>
+            <td>Business analyst at <a href="https://www.creatio.com/" target="_blank">Creatio</a></td>
+        </tr>
+    </table>
+    <div>
+        <p class="line"><strong><u>Skills:</u></strong></p>
```
Comment (RomanIlkiv) [inline lesson_8/index8.html]: tags strong and u already are not needed you know how to make the same by css styles

File: lesson_8/styless8.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    border-bottom: 5px solid salmon;
+    width: 40%;
+}
+img {
+    border: 2px solid sandybrown;
+    padding: 10px;
+}
+table {
+    background-color: rgb(39, 157, 204);
+
```
Comment (RomanIlkiv) [inline lesson_8/styless8.css]: redundant space

File: lesson_8/index8.html (hunk 1 above)
Comment (MasiukBeTech) [reply lesson_8/index8.html]: I did

File: lesson_8/styless8.css (hunk 2 above)
Comment (MasiukBeTech) [reply lesson_8/styless8.css]: I fixed

**Pull request merged**